glusterd2/commands/volumes/volume-reset.go

```diff
@@ -162,5 +162,5 @@ func volumeResetHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	restutils.SendHTTPResponse(ctx, w, http.StatusOK, volinfo.Options)
+	restutils.SendHTTPResponse(ctx, w, http.StatusOK, volinfo)
```

Comment:
Hmm... responding with full volinfo. Sorta makes sense. Should all APIs creating or modifying volumes return volinfos?

Reply:
It does, or at least should.